src/components/Characters.js

```diff
@@ -0,0 +1,57 @@
+import "./Characters.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+export default function Characters() {
+  const [characters, setCharacters] = useState([]);
+  const [page, setPage] = useState(1);
+  const [allPages, setAllPages] = useState();
+
+  useEffect(() => {
+    const url = `https://rickandmortyapi.com/api/character/?page=${page}`;
+
+    fetch(url)
+      .then((res) => res.json())
+      .then((data) => {
+        setCharacters((initialCharacter) => {
+          return [...initialCharacter, ...data.results];
+        });
+        setAllPages(data.info.pages);
+      });
+  }, [page]);
+
+  function handleLoadMore() {
+    if (page < allPages) {
+      setPage(page + 1);
+    }
+  }
+
+  function createCharacters() {
+    return characters.map((character) => {
+      const { id, name, image } = character;
+      return (
+        <div key={id}>
```

Comment:
You could create a CharacterComponent, and use it in createCharacters

Reply:
I know. I didn't dare yet ;)